src/log.h

```diff
@@ -180,6 +181,8 @@ namespace librealsense
 
         void log_to_console(rs2_log_severity min_severity)
         {
+            if( min_severity != RS2_LOG_SEVERITY_NONE )
+                rsutils::os::ensure_console();
```

Comment:
Did you test how python on Windows behave with this code?

Reply:
Everything behaves the same -- python is usually started from a console.
Only thing new is that, if there is no console, a new window will pop up (on Windows).

Reply:
👍

Reply:
I think the current implementation is better.
Let's not surprise the users.